nodejs/awsx/lb/loadBalancer.ts

```diff
@@ -30,6 +30,20 @@ export abstract class LoadBalancer extends pulumi.ComponentResource {
     constructor(type: string, name: string, args: LoadBalancerArgs, opts: pulumi.ComponentResourceOptions) {
         super(type, name, {}, opts);
 
+        if (args.loadBalancer !== undefined) {
+            if (Object.keys(args).length > 1) {
+                throw new pulumi.ResourceError("When providing a loadBalancer argument, no other args are allowed", this);
+            }
+            this.loadBalancer = args.loadBalancer;
+            this.vpc = x.ec2.Vpc.fromExistingIds("lb-vpc", { vpcId: this.loadBalancer.vpcId });
```

Comment:
@leezen does it matter if `lb-vpc` was not the original name of the vpc? This looks a bit surprising to me, but I'm new to Pulumi

Reply:
This is the name of the resource (Pulumi URN) and not the actual VPC name, so it won't matter. However, it'd probably be better if this actually related to the LB name to make it clearer what this resource is, e.g. 
```
`${name}-lb-vpc`
```
or something along those lines.

Reply:
Thanks for clarifying! I like your suggestion, but might tweak it to just `` `${name}-vpc` `` since `name` might already include some test like "lb" or "loadBalancer"